src/BlackSlope.Api.Common/Controllers/BaseController.cs

```diff
@@ -22,7 +22,7 @@ protected ActionResult HandleCreatedResponse(object data, HttpStatusCode status
         protected ActionResult HandleErrorResponse(HttpStatusCode httpStatus, string message)
         {
             var response = new ApiResponse();
-            var errors = new List<ApiError>
+            _ = new List<ApiError>
             {
                 new ApiError
                 {
```

Comment:
We have an open issue[ Fix HandleErrorResponse #17 ](https://github.com/SlalomBuild/blackslope.net/issues/17) . It would be nice if you could fix it as part of this PR.